lib/reporter/index.js

```diff
@@ -323,16 +323,16 @@ class Reporter extends SimplifiedReporter {
   printSimplifiedTestResult(ok, elapsedTime, isWorker) {
     const {currentTest} = this;
 
-    let result = [colors[ok ? 'green': 'red'](Utils.symbols[ok ? 'ok' : 'fail'])];
+    const result = [colors[ok ? 'green': 'red'](Utils.symbols[ok ? 'ok' : 'fail'])];
     if (!this.unitTestsMode) {
       if (isWorker) {
-        result.push(colors.white(process.env.__NIGHTWATCH_ENV, colors.background.black));
+        result.push(colors.white(this.settings.testEnv, colors.background.black));
```

Comment:
Is this change intended?

Reply:
Yes this is intended, we do not pass `__NIGHTWATCH_ENV` with worker task